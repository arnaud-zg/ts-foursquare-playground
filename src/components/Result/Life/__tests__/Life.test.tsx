import React from 'react'
import renderer from 'react-test-renderer'
import { configureStore } from 'ts-foursquare'
import { Life } from '..'
import { Provider } from 'react-redux'

const store = configureStore()

describe('Component/Life', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Life />
        </Provider>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
