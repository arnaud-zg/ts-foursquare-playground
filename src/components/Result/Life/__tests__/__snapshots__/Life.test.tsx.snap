// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component/Life renders correctly 1`] = `
<div>
  <div>
    <h3>
      Life
    </h3>
    <code>
      <pre>
        {"life":false,"credentials":{"clientId":"","clientSecret":""}}
      </pre>
    </code>
  </div>
  <div
    className="mt-4"
  >
    <h3>
      Credentials
    </h3>
    <code>
      <pre>
        {"clientId":"","clientSecret":""}
      </pre>
    </code>
  </div>
</div>
`;
